### ISSUE TITLE
[dataspace] Import September 2023 ETDs

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/dspace-development#783]
The following has been requested on 09/28/23:

> Dear all,
> 
> The September 2023 dissertations and master’s theses have been delivered to ProQuest and will be arriving to one of our servers soon. Could these be imported to DataSpace within three weeks (by October 19)?
> 
> There is just one master’s thesis in this set. This can be published in the Master’s thesis community. It will need a new collection within the community, for the Department of Electrical and Computer Engineering.
> 
> If you have any questions or concerns, please let me know.
> 
> Thank you!
> 
> Lynn

Acceptance criteria:
- [x] Confirm the delivery of ProQuest compressed archives on the PULibrary server infrastructure
- [x] Transfer the delivery of ProQuest compressed archives and generate the ETD import packages using etd_transformer
- [x] Create the Department of Electrical and Computer Engineering Collection within the Master's thesis Community
- [x] Import the DSpace packages into DataSpace
- [x] Confirm that the DSpace packages have been imported into DataSpace

**Comment by bess** [comment]
112 Dissertations imported
> 111 submission_999775
Started: 1696623251225
Ended: 1696623456842
Elapsed time: 205 secs (205617 msecs)

**Comment by bess** [comment]
PhD dissertations imported as expected. Our one master's thesis did not. Tracking down the problem now. :(

**Comment by bess** [comment]
Done. I wrote to Lynn:
> Dear Lynn,
 
I have imported all of the files we received from ProQuest dated 28 September 2023. There were 112 PhD dissertations and 1 Master’s thesis.
 
I created the new department, Electrical and Computer Engineering, under the Master’s Thesis collection. Like last time, the Master’s Thesis is awaiting approval. I’m not sure what’s wrong with the workflow that this keeps happening but I hope you won’t mind going in to approve it.
 
Thanks,
Bess
